goreleaser.sh

```diff
@@ -16,14 +16,17 @@
 GO111MODULE=on
 CGO_LDFLAGS_ALLOW='.*'
 
-Tag=$(git tag --points-at HEAD | grep v[0-9.0-9.0-9])
-FullCommit=$(git rev-parse HEAD)
+# Tries to identify a tag name and defaults to branch name if it fails
+Tag=$(git tag --points-at HEAD | grep "v[0-9]\.[0-9]\.[0-9]*")
+Tag=${Tag:-$(git rev-parse --abbrev-ref HEAD)}
 
-if [[ $(uname) == "Darwin" ]]
+# Assumes anything non-linux has date -u
+Osname=$(uname)
```

Comment:
Why do it need the extra variable?

Reply:
I like the idea. That enables you to do further checks.